src/vcs/git.rs

```diff
@@ -37,15 +37,14 @@
 //! ```
 
 // Re-export git2 as sub-module
-pub use git2::{BranchType, Error as GitError, Oid, Time};
```

Comment:
I would still like to keep these actually. Make it easier for the user since these are directly used in the surf API

Reply:
See 763eace.